Gemfile.lock

```diff
@@ -347,6 +352,7 @@ GEM
     zeitwerk (2.6.12)
 
 PLATFORMS
+  arm64-darwin-23
```

Comment:
This diff is because I'm on a Mac with an M1 chip.

Reply:
Okay